<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doWin</class>
 <widget class="QMainWindow" name="doWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_bt">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cmdStr_l">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="ref_rb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>reference</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">mod_grp</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="im_rb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>import</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">mod_grp</string>
    </attribute>
   </widget>
   <widget class="QScrollArea" name="hold_bx">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>259</width>
       <height>19</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="bx_rf_rb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>reference</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="bx_im_rb">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>import</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="mod_grp"/>
 </buttongroups>
</ui>
